Tasks/AzurePowerShellV4/task.loc.json

```diff
@@ -17,7 +17,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 4,
-    "Minor": 179,
+    "Minor": 180,
```

Comment:
Change the version as mentioned [here](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md)

Reply:
done